zeppelin-integration/src/test/resources/zeppelin-site-star.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0"?>
```

Comment:
where is this file used ?

Reply:
Copied from `zeppelin-server`, removed both unused "zeppelin-site-star.xml" and "zeppelin-site.xml".